ConcernsCaseWork/ConcernsCaseWork.API.Tests/Controllers/DecisionOutcomeControllerTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using ConcernsCaseWork.API.Contracts.Decisions.Outcomes;
+using FluentAssertions;
+using FluentAssertions.Common;
+using Microsoft.AspNetCore.Components.Web;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Mime;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ConcernsCaseWork.API.Tests.Controllers
+{
+	public class DecisionOutcomeControllerTests : IDisposable
```

Comment:
Is this in the right place? The other controller tests in this namespace use mocks. Should this be in the '..tests.integration' namespace?

Reply:
Will look into this as we go forward to see if we can get them stable